mne/preprocessing/ecg.py

```diff
@@ -278,12 +283,13 @@ def create_ecg_epochs(raw, ch_name=None, event_id=999, picks=None, tmin=-0.5,
         and if b is None then b is set to the end of the interval.
         If baseline is equal to (None, None) all the time
         interval is used. If None, no correction is applied.
-    preload : bool
-        Preload epochs or not.
     keep_ecg : bool
         When ECG is synthetically created (after picking), should it be added
         to the epochs? Must be False when synthetic channel is not used.
         Defaults to False.
+    preload : bool
```

Comment:
parameter reordering (see [this](https://github.com/mne-tools/mne-python/pull/5155/files/ff316e16ebf902b970d4bdcd133e69e52bb5885a#r184323650))

Reply:
I didn't mean to reorder these. At one point I was going to get rid of it because it didn't actually work but then put it back, realizing I could make it work properly.

@massich feel free to put it back (keeping the updated description) or I can do it